package.json

```diff
@@ -4,17 +4,19 @@
   "description": "Remix adapter and server runtime for Netlify",
   "main": "index.js",
   "scripts": {
-    "test": "vitest",
+    "build:packages": "pnpm run --filter '@netlify/*' build",
+    "build:packages:watch": "pnpm run --filter '@netlify/*' build:watch",
     "coverage": "vitest run --coverage",
+    "e2e": "pnpm exec playwright test",
     "format": "run-s format:fix:*",
-    "format:ci": "run-s format:check:*",
     "format:check:lint": "eslint --cache .",
-    "format:fix:lint": "eslint --fix --cache .",
     "format:check:prettier": "prettier --check --cache .",
+    "format:check": "run-s format:check:*",
+    "format:fix:lint": "eslint --fix --cache .",
     "format:fix:prettier": "prettier --write --cache .",
-    "e2e": "cypress open",
-    "build:packages": "pnpm run --filter '@netlify/*' build",
-    "build:packages:watch": "pnpm run --filter '@netlify/*' build:watch"
+    "test": "pnpm run typecheck && pnpm run format:check && pnpm run test:unit && pnpm run e2e",
```

Comment:
I personally wouldn't have `typecheck` and `prettier` running as part of the test command. If we want them running in the github actions, I would probably opt to put them in separate lint workflow.

Reply:
Maybe we could leverage `husky` for pre-commit typechecks?

Reply:
I've always thought `test` should test everything and you should have individual scripts you can run for linting, typechecking, unit tests, etc.

To be clear, in CI it's running everything separately. This is just a convenience command for local dev.